upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -404,11 +404,27 @@ func (b *BootstrapChannelBuilder) buildManifest() (*channelsapi.Addons, map[stri
 				Version:           fi.String(version),
 				Selector:          networkingSelector,
 				Manifest:          fi.String(location),
-				KubernetesVersion: ">=1.6.0",
+				KubernetesVersion: "<1.7.0",
+				Id:                id,
+			})
+			manifests[key+"-"+id] = "addons/" + location
+		}
+
+		{
+			location := key + "/k8s-1.7.yaml"
```

Comment:
The main difference for calico is that there are some changes that only work with 1.7 (the NetworkPolicy changes) AFAIK.  I don't think it's incompatible with 1.6 at all though, so it's probably safe to leave out the 1.7 manifest.

At some point I would really like to extract the version requirements for bundled addons like this into the addon manifests, so changes to the channel like this would never be necessary.